packages/table/src/headers/rowHeaderCell.tsx

```diff
@@ -28,6 +31,15 @@ export interface IRowHeaderCellProps extends IProps {
     name?: string;
 
     /**
+     * If true, the row name (string or `ReactElement`) will be replaced with a fixed-height
```

Comment:
remove parenthetical. see corresponding prop docs in file above where you did a great job.